x-pack/plugins/security/server/authentication/api_keys/api_keys.ts

```diff
@@ -8,6 +8,8 @@
 import type { IClusterClient, KibanaRequest, Logger } from '@kbn/core/server';
 
 import type { SecurityLicense } from '../../../common/licensing';
+import type { PutPayloadSchemaType } from '../../routes/authorization/roles/model/put_payload';
+import { transformPutPayloadToElasticsearchRole } from '../../routes/authorization/roles/model/put_payload';
```

Comment:
These should be refactored outside of the `routes` directory, since they're being used for more than routes at this point.